sdk/cosmos/azure-spring-data-cosmos/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### 3.33.0-beta.1 (Unreleased)
 
 #### Features Added
+* Added a new flag `overwritePolicy` to `CosmosIndexingPolicy` that when set to false (by default it is true) will allow the user to prevent the Indexing policy in Portal to be overwritten by the Indexing Policy defined in the SDK  - See [PR 33171](https://github.com/Azure/azure-sdk-for-java/pull/33171)
```

Comment:
Default should be false - also needs to be documented in breaking change section